disnake/ui/button.py

```diff
@@ -293,3 +293,41 @@ def decorator(func: ItemCallbackType) -> DecoratedItem[Button]:
         return func  # type: ignore
 
     return decorator
+
+
+def custom_button(
+    *, cls: Type[B] = Button, **kwargs: Any
+) -> Callable[[ItemCallbackType[B]], DecoratedItem[B]]:
+    """A decorator that attaches a custom button to a component.
+
+    The function being decorated should have three parameters, ``self`` representing
+    the :class:`disnake.ui.View`, the cls being pressed and
```

Comment:
```suggestion
    the :class:`disnake.ui.View`, the cls being passed and
```
same thing in the other decorators